tests/unit/services/job_scheduler/host/test_host.py

```diff
@@ -1,6 +1,8 @@
 from copy import deepcopy
 from itertools import chain
+from mock import call, MagicMock, patch
 import json
+import mock
```

Comment:
It looks like if you add `patch` to the above import, you will not need this line.

Reply:
Done